README.rst

```diff
@@ -39,7 +39,18 @@ You will also need a deployed installation of edX lms and studio to run the test
 See `edx/configuration <http://github.com/edx/configuration>`_ for instructions on provisioning an edX instance.
 For additional information on how to setup your development environment, see `Developer Onboarding <https://openedx.atlassian.net/wiki/pages/viewpage.action?spaceKey=ENG&title=Developer+Onboarding#DeveloperOnboarding-Step4:Getreadytodevelop>`_
 
+OR Using Dockers
+------------
+
+1. git clone https://github.com/edx/edx-e2e-tests
+
+2. docker-compose up --build
+
+3. docker exec -it edxe2etests_e2e-tests_1 /bin/bash
```

Comment:
Where does the name of the container come from? I think you may need to specify it in the docker-compose file